src/main/kotlin/org/rust/cargo/runconfig/buildtool/RsBuildEventsConverter.kt

```diff
@@ -284,8 +282,13 @@ class CargoBuildEventsConverter(private val context: CargoBuildContext) : BuildO
         private val ERROR_OR_WARNING: List<MessageEvent.Kind> =
             listOf(MessageEvent.Kind.ERROR, MessageEvent.Kind.WARNING)
 
-        private fun parseJsonObject(line: String): JsonObject? =
-            JsonParser.parseString(line).takeIf { it.isJsonObject }?.asJsonObject
+        private val StartEvent.taskName: String?
```

Comment:
IMO it'd be great to provide a comment about the structure of `StartEvent` and why it should be trimmed exactly this way

Reply:
I just moved this code from L309-311 to here.